plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -153,10 +153,19 @@ export function handleConfigQuery(
     Array.isArray(queryConfig.values)
   ) {
     try {
-      span.setAttribute(
-        AttributeNames.PG_VALUES,
-        JSON.stringify(queryConfig.values)
-      );
+      const convertedValues = queryConfig.values.map(value => {
+        if (!value) {
+          return 'null';
+        } else if (value instanceof Buffer) {
+          return value.toString();
```

Comment:
maybe use "hex" formatting to get a consistent format.
e.g.
`Buffer.from([65, 0, 1, 2]).toString()` => `'A\x00\x01\x02'`
`Buffer.from([65, 0, 1, 2]).toString("hex")` => `'41000102'`

Reply:
if we use "hex", then the value of `Buffer.from('abc').toString('hex')` is `616263` rather than `abc`. is that what we want to present to the user?

Reply:
not sure what is best here. If buffers actually contain strings using hex encoding is not nice. But if buffers hold binary data it's quite nice. I have no strong opinion here.

By the way, should be also consider some len limit here or just rely on SDK built in limits?

Reply:
1. I left toString() as is. without 'hex'.
2. I added a test for 'toPostgres'.